### DISCUSSION TITLE
Add possibility to declare an implicitly typed local consonants

### DISCUSSION THREAD

**Discussion by DeveloperLookBook** [dotnet/csharplang#6333] [Language Ideas] [answered]
Beginning with C# 3, variables that are declared at method scope can have an implicit "type" var. I propose to extend this idea and to add analog for the constants. 

Syntax could look like this:
void FunctionName(...) 
{
       inv a = 10; // constant declaration. (inv -> invariable)
       // ...
}

Of course you can chose anther name the new keyword ;)

**Comment by HaloFour** [comment, marked as answer]
See: #106
